src/modules/Header/components/Header.js

```diff
@@ -26,23 +30,47 @@ const HeaderBarUpper = styled.div`
   width: 100%;
   background: ${COLORS.WHITE};
   max-height: 50px;
-
 `
 
 const HeaderBarLower = styled.div`
   display: flex;
   justify-content: ${props => props.justifyContent ? props.justifyContent : 'flex-start'};
   margin: 0 auto;
-  padding-left:20px;
   width: 100%;
   background: ${COLORS.OIVA_MENU_BG_COLOR};
   max-height: 50px;
+  overflow: hidden;
+  @media ${MEDIA_QUERIES.MOBILE} {
+    padding-left:32px;
```

Comment:
px units don't scale. It would be better to use em or rem units. Fortunately Tailwind CSS provides suitable units out of the box.